README.md

```diff
@@ -29,6 +29,10 @@ Get the script:
 
          brew install pidcat
 
+If you need to install the latest development version
```

Comment:
This needs indented to match "OS X" above. It needs a colon at the end. And you need an empty line directly following for the next block to be detected as a code block.